.github/ISSUE_TEMPLATE/slack-request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Slack Request
+about: Request a Channel, Bot, Token, or Webhook
+title: 'REQUEST: New Slack <[channel|bot|token|webhook]> <[channel|bot|token|webhook] name>'
+labels: area/community-management, sig/contributor-experience
+assignees: ''
+---
+<!--
+For guidelines on requesting a slack resource, use the below references:
+Channel Request: https://git.k8s.io/community/communication/slack-guidelines.md#Requesting-a-Channel
+Bot/Token/Webhook Request: https://git.k8s.io/community/communication/slack-guidelines.md#Requesting-a-Bot-Token-or-Webhook
+-->
+
+**GitHub Username:**
```

Comment:
Channel/bot/token/webhook owner -- More than likely it will be the same as the requestor, but I can be more explicit about that.

Reply:
> Channel/bot/token/webhook owner -- More than likely it will be the same as the requestor, but I can be more explicit about that.

Looks fine to me as-is...in case the field isn't needed, the requester can probably leave it blank. :woman_shrugging:

Reply:
Added a note about it. 👍